packages/api/lib/granule-remove-from-cmr.js

```diff
@@ -51,12 +51,20 @@ const unpublishGranule = async ({
   pgCollection,
```

Comment:
We should probably mark this as optional in the docstring if it is.

Reply:
That's fair.   Fixed docstring, including defaults.  c1835721b6